testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1978,6 +1980,13 @@ protected void testReadMetadataWithRelationsConcurrentModifications(int readIter
                 future.get(); // non-blocking
             }
         }
+        catch (Throwable failure) {
```

Comment:
The commit message is `Ensure test threads termination even when test overridden` - feels like a general change; but it is for specific test only.

Reply:
wait - did you miss `throw failure` here?

Reply:
> wait - did you miss `throw failure` here?

good catch!



> The commit message is `Ensure test threads termination even when test overridden` - feels like a general change; but it is for specific test only.

yes, but i couldn't put `testReadMetadataWithRelationsConcurrentModifications` in the title line, could i?

Reply:
now that i have read the commit message, i see your point. reworded